src/main/webapp/app/course/manage/course-group.component.ts

```diff
@@ -39,6 +44,7 @@ export class CourseGroupComponent implements OnInit, OnDestroy {
     allCourseGroupUsers: User[] = [];
     filteredUsersSize = 0;
     paramSub: Subscription;
+    exportReady = true;
```

Comment:
I don’t think this variable is ever changed. Can it be removed?

Reply:
Thank you for your comment. You right. I have taken that variable off.